packages/api/src/router/user.ts

```diff
@@ -10,12 +10,20 @@ import { createTRPCRouter, protectedProcedure } from "../trpc";
 export const profileRouter = createTRPCRouter({
   update: protectedProcedure
     .input(
-      z.object({ activeWorkspaceId: z.string().uuid().nullable().optional() }),
+      z.object({
+        activeWorkspaceId: z.string().uuid().nullable().optional(),
+        username: z.string().min(3).max(30).optional(),
```

Comment:
can you create this from the zod drizzle integration so we don't need to manually define it